XamlControlsGallery/XamlControlsGallery.csproj

```diff
@@ -1217,7 +1217,11 @@
     <None Include="testcert.pfx" />
     <None Include="XamlControlsGallery_TemporaryKey.pfx" />
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <SDKReference Include="WindowsDesktop, Version=10.0.18362.0">
+      <Name>Windows Desktop Extensions for the UWP</Name>

```

Comment:
Why is this needed?

Reply:
AppRecordingManager is a desktop only API, but it's also the only thing that captures popups in situ.